FinnUI/Sources/Fullscreen/NotificationCenter/Cells/EmptyNotificationsCell.swift

```diff
@@ -0,0 +1,104 @@
+
+    private lazy var emptyPersonalConstraints = [
+        iconImageView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .spacingM),
+        iconImageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: .spacingM),
+        iconImageView.widthAnchor.constraint(equalToConstant: 80),
+        iconImageView.heightAnchor.constraint(equalToConstant: 80),
+
+        stackView.leadingAnchor.constraint(equalTo: iconImageView.trailingAnchor, constant: .spacingM),
+        stackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: .spacingM),
+        stackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.spacingM),
+
+        contentView.bottomAnchor.constraint(greaterThanOrEqualTo: iconImageView.bottomAnchor, constant: .spacingM),
+        contentView.bottomAnchor.constraint(greaterThanOrEqualTo: stackView.bottomAnchor, constant: .spacingM),
+    ]
+
+    private lazy var emptySavedSearchConstraints = [
+        iconImageView.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+        iconImageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: .spacingXXL),
+        iconImageView.widthAnchor.constraint(equalToConstant: 48),
+        iconImageView.heightAnchor.constraint(equalToConstant: 48),
+
+        stackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .spacingXL),
+        stackView.topAnchor.constraint(equalTo: iconImageView.bottomAnchor, constant: .spacingM),
+        stackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.spacingXL),
+
+        contentView.bottomAnchor.constraint(equalTo: stackView.bottomAnchor, constant: .spacingM),
+    ]
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func configure(with model: EmptyNotificationsCellModel?) {
+        iconImageView.image = model?.icon
+        titleLabel.text = model?.title
+        bodyLabel.text = model?.body
+
+        switch model?.kind {
+        case .personal:
+            iconImageView.layer.cornerRadius = 40
+            stackView.alignment = .leading
+            stackView.spacing = 0
+
+            NSLayoutConstraint.deactivate(emptySavedSearchConstraints)
+            NSLayoutConstraint.activate(emptyPersonalConstraints)
+        case .savedSearch:
+            iconImageView.layer.cornerRadius = 0
+            stackView.alignment = .center
+            stackView.spacing = .spacingL
+
+            NSLayoutConstraint.deactivate(emptyPersonalConstraints)
+            NSLayoutConstraint.activate(emptySavedSearchConstraints)
+        case .none:
+            NSLayoutConstraint.deactivate(emptyPersonalConstraints)
+            NSLayoutConstraint.activate(emptySavedSearchConstraints)
```

Comment:
I guess these constraints are meant to be deactivated?

Reply:
Yes, you're right